src/dymoprint/command_line.py

```diff
@@ -191,6 +197,10 @@ def main():
 
     bitmaps = []
 
+    if args.test_pattern:
+        args.text = " "
```

Comment:
Ya, this is annoying, but not worth properly fixing in this PR. Does `""` work?

Reply:
Ah, nevermind, I see that you're trying to do this as a solution to #88. (I was thinking this was about something else.)

CLI parameter validation happens in the Click library which is pretty idiosyncratic, so don't worry about it.